### PR TITLE
feat(tidb): enable ddlv1 ut pipeline

### PR DESCRIPTION
**Pull request by purelind**
After a period of grayscale testing, we plan to enable this pipeline. This pipeline is set with conditional triggers, and changes to files in the specified directory will trigger it.

### CODE REVIEW

**Comment by purelind** [comment]
/hold

Need more approval  from the relevant team members.

**Comment by purelind** [comment]
image

**Comment by D3Hunter** [comment]
background: we are refactoring DDL job args to be a structured struct, we difference this with previous by a different version, but for compatibility we still need to support V1 for a couple of LTS versions, so we want to run unit tests for both versions, to find bugs earlier

File: prow-jobs/pingcap/tidb/latest-presubmits.yaml [hunk 1]
```diff
@@ -54,10 +54,8 @@ presubmits:
     - name: pingcap/tidb/pull_unit_test_ddlv1
       agent: jenkins
       decorate: false # need add this.
-      # skip_if_only_changed: "(\\.(md|png|jpeg|jpg|gif|svg|pdf)|Dockerfile|OWNERS|OWNERS_ALIASES)$"
-      context: wip/pull-unit-test-ddlv1
-      always_run: false
-      optional: true
+      run_if_changed: "pkg/ddl/.*"
```
Comment (D3Hunter) [inline prow-jobs/pingcap/tidb/latest-presubmits.yaml:57]:
```suggestion
      run_if_changed: "(pkg/|(ddl|meta))/.*"
```

**Review by D3Hunter — APPROVED** [review]

**Review by Benjamin2037 — APPROVED** [review]
LGTM

**Pull request merged**